pkgs/by-name/ad/adbtuifm/package.nix

```
@@ -0,0 +1,29 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  android-tools,
+  lib,
+}:
+buildGoModule rec {
+  pname = "adbtuifm";
+  version = "v0.5.8";
+  src = fetchFromGitHub {
+    owner = "darkhz";
+    repo = pname;
```

use of pname considered anti pattern https://github.com/NixOS/nixpkgs/issues/277994

---

Okay I see, I'll be happy to make that change, but I don't fully understand the differences therer= 
```suggestion
 repo = "adbtuifm"
```

---

the issue is that `pname` may be same ,but not directly related to say "repo"name, eg if someone forks a repo, package `pname` stays same ,but all other references need to be updated.

this may seem like a small nit pick, but for people looking after many packages is a real enough concern

---

That makes sense actually, it is a really small thing but I do see how it can have a sizeable impact on maintaining packages